src/lib/customPropTypes.tsx

```diff
@@ -387,3 +387,5 @@ export const deprecate = (help, validator) => (props, propName, componentName, .
 
   return error
 }
+
+export type ItemShorthandType = React.ReactNode | object | (React.ReactNode | object)[]
```

Comment:
Let's not mix TypeScript typings and PropTypes.  They both shared the name "type" but they are entirely different.

Propose we keep typings in `/types`.